### PR TITLE
add :8080 -> hostname:8080 completion helper

### PR DESCRIPTION
**Pull request by tbg**
completing :8080 to $(hostname):8080 is hopefully a good way of dealing with this issue related
to #662, #672.

also relates to #665 somehow.

### CODE REVIEW

**Comment by tschottdorf** [comment]
(with this change, you can leave out the -gossip and -addr parameters if all you want to do is run a local node for testing and use the KV client)

File: server/server.go [hunk 1]
```diff
@@ -103,7 +94,7 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 
 	s := &Server{
 		ctx:     ctx,
-		host:    host,
+		host:    strings.Split(addr, ":")[0],
```
Comment (petermattis) [inline server/server.go]: Is `Server.host` used anywhere?

**Comment by petermattis** [comment]
LGTM

File: server/server.go (hunk 1 above)
Comment (tschottdorf) [inline server/server.go]: Doesn't seem like it (`git grep '.host' says no). I'll remove it.

**Pull request merged**